cli/README.md

```diff
@@ -0,0 +1,30 @@
+# Alluxio CLI
+
+The Alluxio command line interface is the single entrypoint for users to:
+- Initialize and configure the Alluxio cluster
+- Start and stop processes
+- Expose information about the running cluster
+- Interact with the filesystem, running commands such as `ls` and `cp`
+- Perform administrator level actions, such as format or backup
+
+The CLI is invoked through the shell script at `bin/cli.sh`.
+Commands follow the format of:
+```console
+bin/cli.sh <service> <operation> [--<flag>[=<value>]] [<args>]
+```
+
+Add the `-h` flag to view more details regarding a service or operation.
+
+## Layout and naming conventions
+
+The choice of names for services, operations, and flags should be succinct: short and unambiguous.
+Use of a single word is strongly preferred, but otherwise the name parts should be delimited by a dash such as `foo-bar`.
```

Comment:
wonder if it would be appropriate to declare that flags should be `--foo-bar` but values should be `fooBar`. on one hand, it's more obvious what is a value or argument vs a flag name. on the other hand, it mixes up the casing format.

Reply:
That's a good question. I'm partial to `foo-bar` for both because I don't like typing capital letters in a cli, but it would help visually differentiate them.

Reply:
actually i take that back, we generally don't have a choice in the value's format. that's really up to the underlying implementation...